tests/30rooms/12thirdpartyinvite.pl

```diff
@@ -0,0 +1,419 @@
+use Crypt::NaCl::Sodium;
+use Protocol::Matrix qw( encode_json_for_signing encode_base64_unpadded );
+
+my $crypto_sign = Crypt::NaCl::Sodium->sign;
+
+test "Can invite existing 3pid",
+   requires => [qw( user more_users test_http_server_hostandport )],
+
+   do => sub {
+      my ( $inviter, $more_users, $id_server ) = @_;
+
+      my $invitee_email = "marmosets\@monkeyworld.org";
```

Comment:
FYI, single quotes `''` don't interpolate so this could be simpler

    my $invitee_email = 'marmosets@monkeyworld.org';